frontends/pytorch/csrc/c10_dispatch/acap_dispatch.cpp

```diff
@@ -0,0 +1,82 @@
+//===- acap_dispatch.cpp --------------------------------------------------===//
+//
+// This file is licensed under a pytorch-style license
+// See frontends/pytorch/LICENSE for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "acap_dispatch.h"
+
+#include "npcomp/Python/PybindUtils.h"
+
+#include <c10/core/DispatchKey.h>
+#include <torch/library.h>
+
+using namespace torch_mlir;
+
+namespace py = pybind11;
+
+// TODO: Private use dispatch keys are not made for real uses. Allocate a proper
+// dispatch key in upstream PyTorch (DispatchKey.h) prior to maturity.
+#define ACAP_DISPATCH_KEY PrivateUse1
```

Comment:
I assume that this gets mangled by TORCH_LIBRARY_IMPL into a symbol or something and so needs to have this awkward preprocessor dance? Maybe add a note?

Reply:
Yep. Added a comment.